### PR TITLE
Improved design and usability

### PR DESCRIPTION
**Pull request by Flight**
Improved design and usability of the main table view and popup view

### CODE REVIEW

File: filer/templates/admin/filer/folder/directory_listing.html [hunk 1]
```diff
@@ -94,51 +94,48 @@
...
-            {% if not folder.is_root or folder.is_smart_folder %}
-                <div class="navigator-breadcrumbs">
-                    {% if not folder.is_root %}
-                        <a href="{% url 'admin:filer-directory_listing-root' %}{% if is_popup %}?_popup=1{% if select_folder %}&amp;select_folder=1{% endif %}{% endif %}"
-                            title="{% trans 'Go back to' %} {% trans 'root'|title %} {% trans 'folder' %}">
-                            Filer
+            <div class="navigator-breadcrumbs">
+                <a href="{% url 'admin:filer-directory_listing-root' %}{% if is_popup %}?_popup=1{% if select_folder %}&amp;select_folder=1{% endif %}{% endif %}"
+                    title="{% trans 'Go back to' %} {% trans 'root'|title %} {% trans 'folder' %}">
+                    Root
```
Comment (Brankub) [inline filer/templates/admin/filer/folder/directory_listing.html]: do you need here trans?

Comment (Flight) [inline filer/templates/admin/filer/folder/directory_listing.html]: It was there before, so I believe that yes.

File: filer/static/filer/js/base.js [hunk 2]
```diff
@@ -25,6 +25,15 @@ var Cl = window.Cl || {};
             clickEvent.stopPropagation();
         });
 
+        // Focus on the search field on page load
+        (function () {
+            var filter = $('.js-filter-files');
+
+            if (filter.length) {
+                filter.focus();
```
Comment (vxsx) [inline filer/static/filer/js/base.js]: this would be pretty annoying on touch devices

**Comment by vxsx** [comment]
overall :+1:

**Comment by Brankub** [comment]
:+1:

File: filer/static/filer/js/base.js (hunk 2 above)
Comment (Flight) [inline filer/static/filer/js/base.js]: Will improve it soon. Want to merge it for the demo.

**Pull request merged**